scripts/finish_release.sh

```diff
@@ -20,13 +20,19 @@ if git tag | grep -q "$2" ; then
     exit 1
 fi
 
+CROSS_COMPILE_PROFILE="mac-m1-cross-compile"
+if [ "$ARCH" == "arm" ]; then
```

Comment:
On my machine, `$ARCH` is not defined. I do, however, have the following:

```
% uname -p
arm
```

Reply:
missed to copy some stuff... fixed